packages/flutter_tools/lib/src/artifacts.dart

```diff
@@ -485,7 +465,7 @@ class CachedArtifacts implements Artifacts {
         final String path = _fileSystem.path.join(
           _getFlutterWebSdkPath(),
           'kernel',
-          'amd',
+          'amd-sound',
```

Comment:
I suspect this may be already on your radar, but should we update the generation to exclude `-sound` instead?

https://github.com/flutter/flutter/blob/master/engine/src/flutter/web_sdk/BUILD.gn

Reply:
Ooh, I like that more. Let me tweak a bit.

Reply:
Nice, thanks! I suspect you'll need to change this as well: https://github.com/flutter/flutter/blob/01c28667802ebd82db875d84599f99695524a413/packages/flutter_tools/lib/src/artifacts.dart#L272.